ConwaysGameOfLife/GameController.cs

```diff
@@ -29,34 +29,31 @@ rules checked
         */
         private Universe _universe;
 
-        private IOutput _output;
-
         private IInput _input;
 
-        public GameController(Universe universe, IOutput output, IInput input)
+        private IOutput _output;
+
+        public GameController(IInput input, IOutput output)
         {
-            _universe = universe;
-            _output = output;
             _input = input;
+            _output = output;
         }
 
         public void CreateInitialUniverse()
         {
             _output.WriteLine(Messages.RequestDimensions);
             var input = CreateValidUniverseString();
-            InputParser.ParseUniverse(input);
-            DisplayUniverse();
+            _universe = InputParser.ParseUniverse(input);
         }
 
         public void PopulateUniverseWithLiveCells()
         {
             // _output.Write(Messages.RequestLiveCell);
-            // _output.WriteLine($"Or {Messages.Play});
+            // _output.WriteLine($ or {Messages.Play});
             // loop
             // var locationInput = _input.ReadLine();
-            // var location = SetLiveCellLocation(locationInput, universe.GridWidth, universe.GridLength);
+            // var location = SetLiveCellLocation(locationInput, _universe.GridWidth, _universe.GridLength);

```

Comment:
do you still need the width and length params? or can you access them within the function

```suggestion
            // var location = SetLiveCellLocation(locationInput);
```